docs/helm-chart/parameters-ref.rst

```diff
@@ -480,9 +534,6 @@ The following tables lists the configurable parameters of the Airflow chart and
    * - ``multiNamespaceMode``
      - Whether the KubernetesExecutor can launch pods in multiple namespaces
      - ``1``
-   * - ``serviceAccountAnnottions.*``
-     - Map of annotations for worker, webserver, scheduler kubernetes service accounts
-     - ``{}``
 
```

Comment:
@DerekHeldtWerle It looks like we need related documentation for `jobs`, `redis` and `cleanup` here.